lib/src/expand_text.dart

```diff
@@ -123,18 +125,30 @@ class _ExpandTextState extends State<ExpandText>
     });
   }
 
+  Widget buildText() {
```

Comment:
I wanted to know what's the purpose of the `getMaxLines` method.
Could you explain it to me?